hoodie-client/src/test/java/com/uber/hoodie/TestMergeOnReadTable.java

```diff
@@ -66,20 +88,54 @@
     private HoodieCompactor compactor;
     private FileSystem fs;
 
+    //NOTE : Be careful in using DFS (FileSystem.class) vs LocalFs(RawLocalFileSystem.class)
+    //The implementation and gurantees of many API's differ, for example check rename(src,dst)
+    private static MiniDFSCluster dfsCluster;
```

Comment:
LocalFS does not support appends, all other tests write to the Log file only once and then close it, in these tests I write to the log file multiple times.